### PR TITLE
Added documentation for new badge env variables

### PR DESCRIPTION
**Pull request by harmitgoswami**
Title

### CODE REVIEW

**Review by mathjazz — COMMENTED** [review]
LGTM. The only thing I'd change is that the variables are not really required.

I'm also thinking that we should probably have separate lists for Review Master and Translation Champion lists, but that's a separate issue.

**Comment by harmitgoswami** [comment]
Just out of curiosity, when is a variable considered to be required?

**Comment by mathjazz** [comment]
> Just out of curiosity, when is a variable considered to be required?

When the app breaks if it's not set.

All variables are required, unless the descriptions says they are optional. So please start the descriptions with `Optional.`. Sorry for not making it clearer earlier.

**Review by mathjazz — APPROVED** [review]

File: docs/admin/deployment.rst [hunk 1]
```diff
@@ -72,6 +72,18 @@ you create:
    Set to 'gitlab' if you want to use 'GitLab' (corresponding GITLAB_* settings must be set if required).
    Set to 'google' if you want to use 'Google' (corresponding GOOGLE_* settings must be set).
 
+``BADGES_PROMOTION_THRESHOLDS``
+   Optional. A comma-separated list of numeric thresholds for different levels of the
```
Comment (flodolo) [inline docs/admin/deployment.rst:76]: I'm not sure these explanations are clear without looking at the code.

For each of these options: 
* Do we support any number of levels? 
* Providing an example would also help, maybe with the default value.

File: docs/admin/deployment.rst [hunk 2]
```diff
@@ -72,6 +72,18 @@ you create:
...
 
+``BADGES_PROMOTION_THRESHOLDS``
+   Optional. A comma-separated list of numeric thresholds for different levels of the 
+   Community Builder badge.
+
+``BADGES_START_DATE``
+   Optional. Specifies the start date from which user activities count towards badge achievements. 
+   This variable should be in YYYY-MM-DD format.
+
+``BADGES_TRANSLATION_THRESHOLDS``
```
Comment (flodolo) [inline docs/admin/deployment.rst:83]: Question for a potential follow-up: I'm not sure we should have one setting shared between two different badges.
The numbers happen to match in our proposal, but now we're enforcing that for anyone wanting to use badges.

Comment (flodolo) [reply docs/admin/deployment.rst:83]: Somehow missed this comment

> I'm also thinking that we should probably have separate lists for Review Master and Translation Champion lists, but that's a separate issue.

**Pull request merged**